internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,62 @@
 package daysteps
 
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
 type DaySteps struct {
-	// TODO: добавить поля
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
 }
 
 func (ds *DaySteps) Parse(datastring string) (err error) {
-	// TODO: реализовать функцию
+	parsedStr := strings.Split(datastring, ",")
+	if len(parsedStr) != 2 {
+		return fmt.Errorf("a datastring does not have enough data or too big")
+	}
+
+	ds.Steps, err = strconv.Atoi(parsedStr[0])
+	if err != nil {
+		return err
```

Comment:
Лучше всегда оборачивать ошибки, чтобы было проще потом найти, где она возникла.
```suggestion
		return fmt.Errorf("invalid steps: %w", err)
```